provisioning/mbedtls_user_config.h

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Copyright (C) 2006-2019, Arm Limited, All Rights Reserved
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may
+ *  not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  This file is part of Mbed TLS (https://tls.mbed.org)
+ */
+
+/* Enable PSA APIs, which this example depends on. */
+#define MBEDTLS_PSA_CRYPTO_C
+
+/* Enable PSA use of secure elements. */
+#define MBEDTLS_PSA_CRYPTO_SE_C
+
+/* Make Mbed TLS use PSA. */
```

Comment:
What does that mean? PSA framework? PSA attestation? PSA storage?

> Make Mbed TLS use the PSA Crypto API.

Reply:
Has this been addressed?